lib/iris/analysis/_grid_angles.py

```diff
@@ -65,13 +66,13 @@ def _latlon_from_xyz(xyz):
     Returns:
 
         lonlat : (array)
-            spherical angles, of dims (2, <others>), in radians.
+            spherical angles, of dims (2, <others>), in degrees.
             Dim 0 maps longitude, latitude.
```

Comment:
Here's this `Dim - maps longitude, latitde` again. I don't know what that means?

Reply:
(I do prefer that this is working in degrees)